### DISCUSSION TITLE
Importing Baseline Settings into Intune

### DISCUSSION THREAD

**Discussion by giscus** [ugurkocde/intunemacadmins#50] [General]
# Importing Baseline Settings into Intune

Step-by-step guide to importing baseline settings into your Intune tenant.

https://www.intunemacadmins.com/baselinesettings/import

**Comment by midiman1000** [comment]
Hello, nice work, trouble is I've tested many and verified json formatting, and Intune will not accept them. You may need to look into this further.

**Comment by midiman1000** [comment]
Hello, just to be clear, the native ones work, but the compliance baselines under Intune Management do not, all 3 compliance ones do not work.

**Comment by BenjaminHoegh** [comment]
I'm curious why some of them are device-based while others are user-based. The guide doesn't mention anything about this difference. So my question is: are these OpenIntuneBaseline profiles configured incorrectly, or is the issue with the guidance provided by IntuneMacAdmins?
